src/Components/utils.ts

```diff
@@ -0,0 +1,15 @@
+export function asCssLengthUnit(value?: number | string): string | undefined {
+  if (value === undefined) {
+    return undefined;
+  }
+
+  if (typeof value === "string") {
+    return value;
+  }
+
+  if (typeof value !== "number") {
+    return undefined;
+  }
```

Comment:
When would it get here?

Reply:
Yeah, it'd have to be a value that makes it past the type checker. Happy to take this out if its unnecessary.